polyglot/piranha/src/lib.rs

```diff
@@ -11,6 +11,14 @@ Copyright (c) 2022 Uber Technologies, Inc.
  limitations under the License.
 */
 
+use models::{piranha_arguments::PiranhaArguments, source_code_unit::SourceCodeUnit, piranha_output::PiranhaOutputSummary};
+
+mod config;
+pub mod models;
+#[cfg(test)]
```

Comment:
Just to be clear: this applies only to the next element, right? So, `mod tests;` are only included in the test configuration, but `utilities` are included either way?

Reply:
This annotation applies to the next statement only. 
`mod tests` is used as "test-code" only.
`Utilities` is used as "main-code", and it is also used in "test-code". But because it is used in "main-code" we didnt annotate it.